lib/fir/fir-procedure.h

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2018, NVIDIA CORPORATION.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef FORTRAN_IR_PROCEDURE_H_
+#define FORTRAN_IR_PROCEDURE_H_
+
+namespace Fortran::IR {
+class Procedure;
+struct GraphWriter;
+}
+
+#include "fir-program.h"
+#include "fir-region.h"
+#include "mixin.h"
```

Comment:
`mixin.h` isn't in this PR.

Reply:
Yeah. I'm thinking it's throw-away code.